operator/validator/event_handler.go

```diff
@@ -295,12 +296,12 @@ func (c *controller) handleValidatorRemovedEvent(
 	}
 
 	// remove decided messages
-	messageID := spectypes.NewMsgID(types.GetDefaultDomain(), share.ValidatorPubKey, spectypes.BNRoleAttester)
-	store := c.ibftStorageMap.Get(messageID.GetRoleType())
-	if store != nil {
-		if err := store.CleanAllInstances(logger, messageID[:]); err != nil { // TODO need to delete for multi duty as well
-			return nil, errors.Wrap(err, "could not clean all decided messages")
-		}
+	err := c.ibftStorageMap.Each(func(role spectypes.BeaconRole, store qbftstorage.QBFTStore) error {
+		messageID := spectypes.NewMsgID(types.GetDefaultDomain(), share.ValidatorPubKey, role)
+		return store.CleanAllInstances(logger, messageID[:])
+	})
+	if err != nil {
+		return nil, errors.Wrap(err, "could not clean all decided messages")
```

Comment:
Let's not use `"github.com/pkg/errors"` for new code as it's discontinued
```suggestion
		return nil, fmt.Errorf("clean all decided messages: %w", err)
```